extensions/flyway/deployment/src/test/java/io/quarkus/flyway/test/FlywayExtensionConfigDefaultDataSourceWithoutFlywayTest.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.flyway.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import javax.inject.Inject;
+
+import org.flywaydb.core.Flyway;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+/**
+ * Assures, that flyway can also be used without any configuration,
+ * provided, that at least a datasource is configured.
+ * 
+ * @author JohT
```

Comment:
Global remark: as explained above, let's get rid of the `@author` tags. We rely on git history for attribution.

Reply:
ok